src/foam/nanos/app/AppConfig.js

```diff
@@ -63,6 +63,11 @@ foam.CLASS({
       class: 'String',
       name: 'playLink',
       value: 'https://play.google.com/store?hl=en'
+    },
+    {
+      class: 'Boolean',
+      name: 'enableForceHttps',
```

Comment:
rename to forceHttps
But this can wait for the next PR, we need to get this out